onnxruntime/core/providers/cpu/tensor/identity_op.cc

```diff
@@ -8,19 +8,20 @@ namespace onnxruntime {
 ONNX_CPU_OPERATOR_VERSIONED_KERNEL(
     Dropout,
     7, 9,
-    KernelDefBuilder().TypeConstraint("T", {DataTypeImpl::GetTensorType<MLFloat16>(), 
-                                            DataTypeImpl::GetTensorType<float>(), 
+    KernelDefBuilder().TypeConstraint("T", {DataTypeImpl::GetTensorType<MLFloat16>(),
+                                            DataTypeImpl::GetTensorType<float>(),
                                             DataTypeImpl::GetTensorType<double>()})
-                      .TypeConstraint("T1", DataTypeImpl::GetTensorType<bool>()),
+        .TypeConstraint("T1", DataTypeImpl::GetTensorType<bool>()),
     IdentityOp<true>);
 
-ONNX_CPU_OPERATOR_KERNEL(
+ONNX_CPU_OPERATOR_VERSIONED_KERNEL(
     Dropout,
     10,
+    11,
```

Comment:
What about Dropout - 12 ?

Reply:
It's in a separate file now. nn/dropout_op.cc/h